internal/controller/constants.go

```diff
@@ -4,6 +4,8 @@ const (
 	/*** Operator Settings ***/
 	// OLSConfigName is the name of the OLSConfig Custom Resource
 	OLSConfigName = "cluster"
+	// DefaultReconcileInterval is the default interval for reconciliation
+	DefaultReconcileInterval = 300
```

Comment:
i think something more like 2 hours is typical for relisting/forcing reconciliation.  Did you base this value on another operator?

Reply:
We only had this as starting point. I am good with 2 hours - I will udpate